src/Schemas/MetadataSchema.php

```diff
@@ -0,0 +1,207 @@
+<?php
```

Comment:
I think hardcoding metadata isn't a good idea. You should generate it from external data.